academic-services/knowledge-exploration-service/TOC.yml

```diff
@@ -46,6 +46,8 @@
     href: tutorial-entity-linking.md
   - name: Build browser applications with contextual filters
     href: tutorial-schema-design.md
+  - name: Add search to the library browser
```

Comment:
>Add search to the library browser [](start = 10, length = 33)

This name doesn't fit the others in the group. Maybe "Build browser applications with search"?